services/BUILD.bazel

```diff
@@ -0,0 +1,24 @@
+java_library(
+    name = "services",
+    srcs = glob([
+        "src/main/java/io/grpc/**/*.java",
+        "src/generated/main/**/*.java",
```

Comment:
We usually dont link to generated sources, since bazel usually  generates the the source ephemerally.

Reply:
Good point. I added proto_library, java_proto_library, and java_grpc_library targets in c91d2004b49e2f3262abb0ac843c004f0bbfb6c8, so that the top-level services targets can depend on them instead of the gencode. PTAL.